test/groovy/src/com/puzzleitc/jenkins/QuaySpec.groovy

```diff
@@ -0,0 +1,56 @@
+package groovy.src.com.puzzleitc.jenkins
+
+import com.puzzleitc.jenkins.Quay
+
+import static org.assertj.core.api.Assertions.*
+
+import com.lesfurets.jenkins.unit.BasePipelineTest
+import org.junit.Before
+import org.junit.Test
+
+/**
+ * Tests for vars/check.groovy
+ */
+class QuaySpec extends BasePipelineTest {
```

Comment:
I would suggest using `QuayTest` instead of `QuayTests` so we have more differentiation between Spock and JUnit.

Reply:
agree